client/src/customer-pages/pages/MapTap/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+import { NavBar, Tabs, DetailsCard } from "../../../components";
+import "./map.css";
+
+class MapTap extends Component {
+  render() {
+    return (
+      <div className="map-container">
```

Comment:
If you're interested, this is another place where you could apply some of these ideas (#25) to put the common code between MapTap and FlightRoute Tap (basically the NavBar and Tabs) into a single file.  It would probably require some adjustments to how the classes are currently passed into the Tabs component though.